packages/sparrowql/package.json

```diff
@@ -14,5 +14,8 @@
   ],
   "engines": {
     "node": ">=8"
+  },
+  "devDependencies": {
+    "@types/mongodb": "^3.6.3"
```

Comment:
1. Pin the version.
2. If you'd add it to the top `package.json` as well, it won't create a separate `package-lock.json`.